packages/caliper-fisco-bcos/package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@hyperledger/caliper-fisco-bcos",
+    "version": "0.1.0",
+    "description": "Caliper adapter of FISCO BCOS blockchain backend",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/vita-dounai/caliper.git"
+    },
+    "engineStrict": true,
+    "engines": {
+        "node": ">=8.10.0",
+        "npm": ">=5.6.0"
+    },
+    "files": [
+        "*",
+        "src/comm/template/report.html"
+    ],
+    "dependencies": {
+        "caliper-core": "0.1.0",
+        "nconf": "^0.10.0",
+        "isarray": "^2.0.4",
+        "web3": "0.20.7",
+        "fs-extra": "^8.0.1",
+        "bn.js": "^4.11.8",
+        "ethereumjs-util": "^6.1.0",
+        "crypto-js": "^3.1.9-1",
+        "ora": "^1.2.0",
+        "uuid": "^3.3.2",
+        "tls": "^0.0.1",
+        "chalk": "^1.1.3",
+        "request-promise": "^4.2.1"
+    },
+    "nyc": {
+        "exclude": [
+            "lib/**"
+        ],
+        "reporter": [
+            "text-summary",
+            "html"
+        ],
+        "all": true,
+        "check-coverage": false,
+        "statements": 5,
+        "branches": 8,
+        "functions": 7,
+        "lines": 5
+    },
```

Comment:
We are also including a license checker now, to ensure that all files have the correct license during the CI build process. This is performed at the package level, with the check being started as a `pre test` phase. Would you be willing to include/use the same settings as the other packages?

Reply:
test is valid now